R/calculateDMM.R

```diff
@@ -257,9 +257,11 @@ setMethod("plotDMNFit", signature = c(x = "SummarizedExperiment"),
         k <- vapply(dmn, function(d){ncol(mixture(d))}, numeric(1))
         fit <- vapply(dmn, fit_FUN, numeric(1))
         ggplot(data.frame(k = k, fit = fit), aes_string(x = k, y = fit)) +
-          geom_point() +
-          geom_line() +
-          theme_bw()
+            geom_point() +
+            geom_line() +
+            theme_bw() +
+            xlab("Number of Dirichlet Components") +
+            ylab(paste0("Model Fit (",type,")"))
```

Comment:
I suggest using the `labs` function i.e. `labs(x = "Number of Dirichlet Components", y = paste0("Model Fit (",type,")"))` instead of `xlab` and `ylab`. "The labs function is preferable to xlab/ylab/ggtitle, since it generalizes to other aesthetics and labels (e.g. subtitle and caption), which don't have their own functions."